### DISCUSSION TITLE
Preview release [22.05]

### DISCUSSION THREAD

**Discussion by isublimity** [tabixio/tabix#241] [Show and tell]
For Bugs & Ideas

**Comment by leiting6** [comment]
There is an issue, blow question:

#docker-compose up -d
Building react
Step 1/6 : FROM node:10
 ---> 28dca6642db8
Step 2/6 : WORKDIR /usr/src/app/react
 ---> Using cache
 ---> 0c29da756c5a
Step 3/6 : COPY package*.json ./
ERROR: Service 'react' failed to build : COPY failed: no source files were specified

**Comment by romkoval** [comment]
Привет! почему отключили draw панель?
